src/About/style.scss

```diff
@@ -0,0 +1,96 @@
+@import './_variables.scss';
+
+.wrapper {
+  background-attachment: fixed;
+  background-image: url('../image/bg_about.png');
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.section-about {
+  display: flex;
+  width: 70%;
+  padding: 60px 0 20px;
+  margin: 0 auto;
+}
+
+.about-right {
+  margin-left: 5%;
+}
+
+.about-left {
+  margin-top: 35px;
+  margin-left: 11%;
+
+
+  h1 {
+    padding-bottom: 13px;
+    font-size: 36px;
+    color: $budha-gold;
+    text-align: center;
+  }
+
+  p {
+    margin: 30px 0 50px;
+    font-size: 16px;
+    line-height: 2;
+    text-align: center;
+  }
+}
+
+
+.mw-100 {
+  min-width: 100%;
+}
+
+img {
+  margin-left: 0;
+  text-align: center;
+}
+
+.cook {
+  padding-top: 18px;
+  margin-left: 33%;
+}
+@media screen and (max-width: 1414px) {
```

Comment:
pls use var
also pls use more general breakpoints

Reply:
i changed  all resolution to variable